movie/src/components/Card/index.jsx

```diff
@@ -41,15 +41,23 @@ const Card = ({ movies }) => {
   };
 
   // use on style.
-  const isCircleColor = (voteAverage) =>
-    voteAverage > 80 ? "success" : "warning";
+  const getCircleColor = (voteAverage) => {
```

Comment:
Could you double-check this function? You haven’t covered the case between `31-71` 😅